docs/content/Hidden.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Hidden
+description: Use Hidden to responsively hide or show content in narrow, regular and wide viewports.
+source: https://github.com/primer/react/blob/main/src/Hidden/index.tsx
+status: Alpha
+componentId: hidden
+---
+
+The `Hidden` component is a `<span />` that will help you hide or show content/components in different viewports.
+**Attention:** Use Hidden only to control behaviour in a responsive manner. It does not take any `sx` props.
+
+## Example
+
+```jsx live
+<Hidden when="narrow">
+  <Placeholder height="80px" label="This is not visible in narrow viewport" />
+</Hidden>
+```
+
+## Array as `when` prop
+
+```jsx live
+<Hidden when={['narrow', 'wide']}>
+  <Placeholder height="80px" label="This is not visible in narrow and wide viewport" />
+</Hidden>
+```
+
+## Props
+
+<PropsTable>
+  <PropsTableRow
+    name="when"
+    type="string or Array"
+    description="Can be one or more values of 'narrow', 'wide', 'regular'"
```

Comment:
Wonder if a more specific definition would be helpful here:
```suggestion
    type={`
| 'narrow'
| 'regular'
| 'wide'
| Array<'narrow' | 'regular' | 'wide'>`}
  description="The viewport range(s) at which children should be hidden"
```

Reply:
Yeah, I could write the actual size values.